test/pickWithDefault.js

```diff
@@ -0,0 +1,20 @@
+var R = require('..');
+var eq = require('./shared/eq');
+
+
+describe('pickWithDefault', function() {
+  var obj = {a: 1, b: 2, c: 3, d: 4, e: 5, f: 6};
+  it('copies the named properties of an object to the new object', function() {
+    eq(R.pickWithDefault(['a', 'c', 'f'], null, obj), {a: 1, c: 3, f: 6});
+  });
+
+  it('defaults properties not present on the input object', function() {
+    eq(R.pickWithDefault(['a', 'c', 'g'], null, obj), {a: 1, c: 3, g: null});
+  });
+
+  it('is curried', function() {
+    var copyAdefaultG = R.pickWithDefault(['a', 'g']);
+    eq(copyAdefaultG(null, obj), {a: 1, g: null});
+  });
```

Comment:
We could remove this test. All functions in the library are curried, and we already have a test which asserts this.